components/mm-components/spinner/MmSpinner.vue

```diff
@@ -1,56 +1,23 @@
 <template>
-    <div role="spinner" aria-valuemin="0" aria-valuemax="100" :aria-valuenow="normalizedValue">
-        <svg class="spinner" xmlns="http://www.w3.org/2000/svg" viewBox="23 23 48 48" style="transform: rotate(-90deg);">
-            <circle fill="transparent" cx="48" cy="48" r="20" stroke-width="3" :stroke-dasharray="strokeDashArray"
+    <div role="spinner" aria-valuemin="0" aria-valuemax="100">
+        <svg class="spinner-svg" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 52 52"
+            style="transform: rotate(-90deg);">
+            <circle fill="transparent" cx="26" cy="26" r="24" stroke-width="3" stroke-dasharray="157"
                 stroke-dashoffset="0" class="spinner-underlay"></circle>
-            <circle fill="transparent" cx="48" cy="48" r="20" stroke-width="3" :stroke-dasharray="strokeDashArray"
-                :stroke-dashoffset="strokeDashOffset" class="spinner-overlay"></circle>
+            <circle fill="transparent" cx="26" cy="26" r="24" stroke-width="3" stroke-dasharray="157"
+                stroke-dashoffset="157" class="spinner-overlay"></circle>
         </svg>
     </div>
 </template>
 
 <script>
-export default {
-    props: {
-        value: {
-            type: [Number, String],
-            default: 0
-        },
-        loading: {
-            type: Boolean,
-            default: false
-        }
-    },
-    data() {
-        return {
-            radius: 20,
-        }
-    },
-    computed: {
-        circumference() {
-            return 2 * Math.PI * this.radius
-        },
-        normalizedValue() {
-            if (this.value < 0) return 0
-            if (this.value > 100) return 100
-
-            return parseFloat(this.value)
-        },
-        strokeDashArray() {
-            return Math.round(Math.round(this.circumference * 1000) / 1000)
-        },
-        strokeDashOffset() {
-            return `${Math.round(((100 - this.normalizedValue) / 100) * this.circumference)}px`
-        }
-    }
-}
+export default {}
 </script>
 
 <style scoped>
-
-.spinner {
-    height: 100px;
-    width: 100px;
+.spinner-svg {
```

Comment:
The spinner should be resizable. It will  need a prop for width only. You will only need to change the width of the svg. 
The best way to do this is assign a css var for the width and  set it on mounted with js.  Here's some psuedo code to point you in the right direction.
```
props: {
    width: String
}

mounted(){
    if(this.width) {
        this.$refs.spinnerSVG.styles.setProperty('spinner-width', this.width);
    }
}

<style>
.spinner-svg {
    --spinner-width: 50px;
    width: var(--spinner-width);
}
</style>
```